src/dialogs/more-info/controls/more-info-update.ts

```diff
@@ -164,7 +135,38 @@ class MoreInfoUpdate extends LitElement {
             `
           : ""}
       </div>
-    `;
+
+      ${supportsFeature(this.stateObj!, UpdateEntityFeature.RELEASE_NOTES) &&
+      !this._error
+        ? this._releaseNotes === undefined
+          ? html`<hr />
+              <div class="flex center">
+                <ha-circular-progress indeterminate></ha-circular-progress>
+              </div>`
+          : html`<hr />
+              <ha-faded>
+                <ha-markdown .content=${this._releaseNotes}></ha-markdown>
+              </ha-faded> `
+        : this.stateObj.attributes.release_summary
+          ? html`<hr />
+              <ha-markdown
+                .content=${this.stateObj.attributes.release_summary}
+              ></ha-markdown>`
+          : ""}
+      ${supportsFeature(this.stateObj, UpdateEntityFeature.BACKUP)
+        ? html`<hr />
+            <ha-formfield
+              .label=${this.hass.localize(
+                "ui.dialogs.more_info_control.update.create_backup"
+              )}
+            >
+              <ha-checkbox
+                checked
+                .disabled=${updateIsInstalling(this.stateObj)}
+              ></ha-checkbox>
+            </ha-formfield> `
+        : ""}
+      <hr />`;
```

Comment:
This should not be moved as it is related to the action! I guess there is no movement to be expected here.

Reply:
This is just how git is showing the diff since the actions got moved up. At least I think that's what your concern is.

Reply:
No, when you have an update of an addon, HA offers to do a backup before. This is changeable via a checkbox. This checkbox was above the buttons before. Now the buttons got moved, but the checkbox will be kind of hidden at the bottom of the page below the changelog and is somewhat disjunct.